.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+repos:
+      - repo: https://github.com/pre-commit/mirrors-clang-format
+        rev: v11.1.0
+        hooks:
+              - id: clang-format
+                files: \.(cu|cuh|h|hpp|cpp|inl)$
+                types_or: [file]
+                args: ['-fallback-style=none', '-style=file', '-i']
```

Comment:
```suggestion
                args: ['-fallback-style=none', '-style=file:thirdparty/cudf/cpp/.clang-format', '-i']
```

Reply:
I was able to do this, but I had to update the clang-format version. In theory this doesn't matter, but I wanted to mention it.